sp/src/game/server/hl2/npc_combine.cpp

```diff
@@ -4215,6 +4300,14 @@ void CNPC_Combine::SpeakSentence( int sentenceType )
 #endif
 		}
 		break;
+
+#ifdef EZ2
+	case 6: // Ordering enemy to surrender
+		SpeakIfAllowed( TLK_USE, "order_surrender:1" );
```

Comment:
Should they still speak if GetEnemy() is null?

Reply:
Soldiers shouldn't be running this schedule at all if the enemy is null, but I think they probably can if the enemy dies mid-surrender, so I added interrupt conditions for that to the schedule itself.